lib/bump/bump.ts

```diff
@@ -71,9 +77,30 @@ export interface AutoBumpOptions {
    * The name of the branch to push the bump commit (e.g. "master")
    * This branch has to exist.
```

Comment:
This can't be true if we're making a PR.

Reply:
I'll add a little check that forbids this case.